quad_manipulator:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  arm_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_joint_1
    pid: {p: 1, i: 0.05, d: 0.001} # pid: {p: 10.0, i: 0.5, d: 0.01}
  arm_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_joint_2
    pid: {p: 1, i: 0.05, d: 0.001} # pid: {p: 80.0, i: 1.0, d: 0.01}
  arm_joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_joint_3
    pid: {p: 1, i: 0.05, d: 0.001} # pid: {p: 10.0, i: 0.5, d: 0.01}
  grasp_joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: grasp_joint_1
    pid: {p: 0.5, i: 0.05, d: 0.001} # pid: {p: 10.0, i: 0.5, d: 0.1}
  grasp_joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: grasp_joint_2
    pid: {p: 0.5, i: 0.05, d: 0.001} #pid: {p: 0.5, i: 0.05, d: 0.001}
